.env

```diff
@@ -0,0 +1,83 @@
+#JPO ODE Docker environment file
```

Comment:
Lets remove the comments per line since they are duplicates of the wiki guide anyway.